action/consignment_transfer.go

```diff
@@ -46,18 +49,25 @@ type (
 
 	// ConsignMsg is the consignment message
 	ConsignMsg struct {
-		Index     int    `json:"index"`
+		Type      string `json:"type"`
```

Comment:
add type of signature/chain, per Zhi's comment 1

Reply:
"Type" is supposed to be a member of ConsignJSON.